features-json/webxr.json

```diff
@@ -350,9 +350,10 @@
   },
   "notes":"",
   "notes_by_num":{
-    "1":"Chrome 66-72 supported a very early version of the spec",
-    "2":"Part of an origin trial",
-    "3":"Part of the spec can be enabled with the [webxr](chrome://flags/#webxr) and [webxr-hit-test](chrome://flags/#webxr-hit-test) flags."
+    "1":"Chrome 66-72 supported a very early version of the spec.",
+    "2":"Part of an origin trial.",
```

Comment:
I wouldn't mention the origin trial either. Chrome 78 is the last version for it. Participating in an OT requires a sign-up and an origin-specific token. [All tokens will be deactivated on December 3.](https://developers.chrome.com/origintrials/#/view_trial/-5670031930859454463) After that date there will be no way to use XR in versions 76, 77, or 78.